adsft/tests/test_extraction.py

```diff
@@ -52,10 +53,15 @@ def test_that_we_can_parse_the_xml_content(self):
         """
 
         full_text_content = self.extractor.open_xml()
-        for parser_name in ("html5lib", "html.parser", "lxml-html", "direct-lxml-html", "lxml-xml", "direct-lxml-xml",):
-            self.extractor.parse_xml(preferred_parser_names=(parser_name,))
-            journal_title = self.extractor.extract_string('//journal-title')
 
+        if self.parsers:
+            for parser_name in self.parsers:
+                self.extractor.parse_xml(preferred_parser_names=(parser_name,))
+                journal_title = self.extractor.extract_string('//journal-title')
+                self.assertEqual(journal_title, 'JOURNAL TITLE')
+        else:
+            self.extractor.parse_xml()
+            journal_title = self.extractor.extract_string('//journal-title')
```

Comment:
I do not understand why we would need to have this `else` in every single unit test. I understand it could help us catch a mistake in the default argument of the `parse_xml` function, but we could have one single unit test to test that if it is necessary. Doing this in every unit test complicates the code and it won't get executed unless the developer sets `self.parsers` to None. Or maybe I am missing something else?